src/app/crypto/services/pgp.service.ts

```diff
@@ -57,9 +59,6 @@ export const encryptMessageWithPublicKey = async ({
   publicKeyInBase64: string;
   publicKyberKeyBase64: string;
 }): Promise<string> => {
-  if (message.length != 32) {
```

Comment:
@sg-gs I had to remove the 256-bit restriction on input messages. 

I was wrong to assume that a mnemonic is 256 bits. It's a string of much larger and varied lengths. So, I use hashing to expand the Kyber secret to whatever size we need for XORing. This is what the original Kyber paper does, but since NIST prefers a fixed-length output they removed it.

Kyber authors used sha3 but also suggested Blake as a faster alternative. I use Blake3 here only because the library allows any output length for blake3 and only a few options for sha3 output (224, 256, 384, and 512).